source/includes/getting-started/_install.html.md

```diff
@@ -0,0 +1,181 @@
+
+The next section walks you through how to install Python, Git, and Pip, but if you're already ahead of the curve and have them all installed, skip down to [Setup the Bonsai CLI][2]. 
+
+**Note:** We highly recommend Mac users use [HomeBrew][6] or download Python directly in the General Install instructions so they can perform the following steps on user-land Python rather than [system Python][13].
+
+**If you have Anaconda installed on your system, skip down to [Install with Anaconda][9].**
+
+If you *do not* have Anaconda installed on your system continue to follow our **General Install** instructions.
+
+## General Install
+
+### Windows
+
+> [Chocolatey][14] commands
+
+```shell
+choco install git
+choco install python # or you can install python2 if you need version 2.7
+```
+
+1. **Git**: Download and install Git from [Git for Windows][11] or with [Chocolatey][14].
+2. **Python**: Download and install Python from [python.org][3] or with [Chocolatey][14].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+**Note:** You may need to reboot your computer after installing Python and Git to make sure all install settings take effect.
+
+### Mac OS X
+
+> [Homebrew][6] commands
+
+```shell
+brew install git
+brew install python3 # or you can install python if you need version 2.7
+```
+
+1. **Git**: Git is more than likely already installed on your computer if you have Xcode, but if not, run the git command or install with [Homebrew][6].
+2. **Python**: Download and install Python from [python.org][3] or with [Homebrew][6].
+3. **Pip**: Python version 2.7.9 and greater come with pip, but if for some reason you need to use a different version of python please follow [these instructions][12].
+
+## Install with Anaconda
+
+If you *do not* have Anaconda installed on your system, please skip this section.
+
+conda is a tool for installing and managing Python and R dependencies. For more information, refer to the [Anaconda website][10].
+
+> Windows
+
+```shell
+conda create -n gym-env
+activate gym-env
+conda install pip
+```
+
+> Mac OS
+
+```shell
+conda create -n gym-env
+source activate gym-env
+conda install pip
+```
+
+# Setup the Bonsai CLI
+
+Install the Bonsai Command Line Interface tool and configure it. You'll need to have your access code (accessKey). You can find your access code in your [Bonsai Account Settings][8].
+
+```shell
+pip install bonsai-cli
+bonsai configure # Follow the instructions to retrieve your key, and enter it when prompted
+```
+
+## Clone a sample project
+
+Clone a sample simulation project with git. For this guide we'll be walking you through [OpenAI Gym's][4] Mountain Car simulation.
```

Comment:
Keen and I agreed to mention that an OpenAI gym environment is a type of simulator in the docs.